index.html

```diff
@@ -204,28 +221,104 @@ <h2 class="text-[32px] text-dark-blue font-bold mt-3 -mb-2">
           >
         </div>
       </div>
-      <!--* end project one -->
+      <!--! end project one -->
 
-      <!--* project two -->
+      <!--!! project one -- md: number three -->
       <div
-        class="w-[100%] p-4 border border-light-grey-2 bg-white rounded-2xl flex flex-col justify-center items-center"
+        class="w-[100%] p-4 border md:order-3 md:h-[496px] md:gap-x-[2.9rem] border-light-grey-2 bg-white rounded-2xl flex flex-col md:flex-row justify-center items-center"
       >
         <!--* project screen shot -->
-        <div class="w-full h-[50%] bg-white">
+        <div class="w-full md:w-[50%] h-[50%] md:h-full bg-white">
           <img
             class="w-full h-full object-contain"
-            src="./img/projects/project-2.png"
+            src="./img/desktop/desktop-project-3.jpg"
             alt="project-1"
           />
         </div>
         <!--* project info -->
-        <div class="flex flex-col justify-around items-start gap-y-4">
-          <h2 class="text-[32px] text-dark-blue font-bold mt-3 -mb-2">
+        <div
+          class="flex md:w-[50%] md:h-full flex-col justify-around md:justify-start items-start gap-y-4"
+        >
```

Comment:
- [ ] I think it would be better to replicate[ figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=lm66x04u5euYTBoh-0) on element spacing of your contents. It would be better to use the same CSS padding for ```top, right, left and bottom``` to create same space around the element, inside any defined border. :joy: 

_Your may follow this [link ](https://www.geeksforgeeks.org/css-margins-padding/)for more info :+1: :+1:_ 

**Your design**
![image](https://user-images.githubusercontent.com/102757126/206837250-38364a10-6865-4bbd-9d8f-230c3ab76a83.png)